cypress/integration/not_logged_in.js

```diff
@@ -0,0 +1,17 @@
+describe('Not logged in actions', function () {
+  beforeEach(function () {
+    // before each test, we can automatically preserve the
+    // 'session_id' and 'remember_token' cookies. this means they
+    // will not be cleared before the NEXT test starts.
+    //
+    // the name of your cookies will likely be different
+    // this is just a simple example
+    Cypress.Cookies.debug(true)
+    Cypress.Cookies.preserveOnce('_elixirvictoria_key')
+  })
+
+  it('Visits the main page', function () {
+    cy.visit('http://localhost:5000')
+    cy.contains('Welcome to Phoenix!')
+  })
```

Comment:
About as basic a test as it can be. We can just keep adding js files in this folder and it will run them all.

[The documentation for Cypress is excellent](https://docs.cypress.io/api/api/table-of-contents.html)